src/Features/Core/Portable/Common/Glyph.cs

```diff
@@ -4,6 +4,8 @@ namespace Microsoft.CodeAnalysis
 {
     internal enum Glyph
     {
+        None,
+
```

Comment:
As a reminder, these values get baked into assemblies so if we have anybody else other than TypeScript doing insertions, this will break them.